pkg/rancher-desktop/assets/translations/en-us.yaml

```
@@ -272,6 +276,8 @@ k8s:
     cancel: Cancel
 portForwarding:
   title: Port Forwarding
+  sortableTables:
+    noRows: There are no port forwarding to show
```

This sounds a bit weird to me. "Port forwardings" would at least be plural, but still doesn't sound right. How about "port forwarding entries"?

```suggestion
    noRows: There are no port forwarding entries to show
```

---

Yea, check spelling pipe flagged this, agree with the suggestion